### DISCUSSION TITLE
Check lint for specific node version

### DISCUSSION THREAD

**Discussion by kiranparajuli589** [eslint/eslint#16176] [Q&A] [answered]
Hello, greetings.

I'm using a project that uses NodeJS v14.x.x. Can I use ESLint targeting NodeJS 14 and detect all unresolved functions that are not yet available in v14? For eg, `replaceAll` fn is introduced only in v15. If there are such usages inside, can ESLint help me find such a piece of code?

I have a GitHub workflow demonstrating this [here](https://github.com/kiranparajuli589/node-check/runs/7573211393?check_suite_focus=true).

Thank you.

**Comment by aladdin-add** [comment, marked as answer]
eslint does not support it, but you can checkout a plugin: [eslint-plugin-n](https://www.npmjs.com/package/eslint-plugin-n)..
